### PR TITLE
Various linkifyURLs/Issues improvements and bugfixes

### LINKED ISSUE #132: Links to issue/PR in titles from #61 not always present
The change introduced in #61 is not always working for me, but I can reproduce only sporadically. At first I thought I was going crazy, but it's been 3 days now and I see it a few times a day. I thought it was from back-and-forward navigation, but it's not, I thought it was when loading a page in the background, but it's not. It's something else which prevents it from modifying the title in some cases.

Have you experienced this?

### LINKED ISSUE #484: Linkify URLs in markdown code blocks
URLs should also be linkified in markdown code blocks like these:
```markdown
[That URL should be a link](https://some-url.com/)
```
```js
// This one too: https://some-url.com/
```

### LINKED ISSUE #511: Support & in linkified URLs
`linkifyCode` passes HTML to the linkifier (because it musts: #510), but this means that `&` is first escaped and this causes some URls to break. Example:

```js
"<img src='https://avatars1.githubusercontent.com/u/1402241?v=3&;s=40'>"
//                               The parameters should be: ?v=3&s=40
```

See it here in action: https://github.com/bfred-it/sandbox/blob/master/index.js

A solution I could think of would be to have the linkifier return actual DOM nodes so we don't have to inject plain HTML **and** this means we can pass unescaped data to it.

### LINKED ISSUE #513: Phantom semicolon added to closing XML element
Seen in the wild at https://github.com/thomaslevesque/FakeItEasy/blob/19f6cbcd444c443949eb3a9f43ce52d4eebb65e4/src/FakeItEasy/FakeItEasy.csproj#L18 when using Refined GitHub 1.19.0, it looks like

```xml
<PackageProjectUrl>https://fakeiteasy.github.io/<;/PackageProjectUrl>
```

Here's an image, just in case:

image

Note that the semicolon is not really there, as can be seen in the raw view.

### PR DESCRIPTION
**Pull request by bfred-it**
Depends on: https://github.com/sindresorhus/linkify-issues/pull/1

Fixes #511 
Fixes #513 
Fixes #132 (6764548)
Closes #484 (554e2e5)

### CODE REVIEW

**Comment by bfred-it** [comment]
Try it in previously-supported scenarios:
* Code: [sindresorhus/refined-github/test/linkify-urls-in-code.js
](https://github.com/sindresorhus/refined-github/blob/be756e68dd4bf24d53e802e1fec126506979fa80/test/linkify-urls-in-code.js)
* Code with duplicate URLs: https://github.com/sindresorhus/caprine/pull/220/files
* Title: https://github.com/sindresorhus/refined-github/pull/299
* URL next to HTML open tag: https://github.com/binarykitchen/videomail-client/blob/feature/rollupjs/package.json

And see each of the fixed issues to see more.

File: package.json [hunk 1]
```diff
@@ -16,8 +16,8 @@
 		"dom-loaded": "^1.0.0",
 		"element-ready": "^2.0.0",
 		"github-injection": "^0.3.0",
-		"linkify-issues": "^1.1.0",
-		"linkify-urls": "^1.0.2",
+		"linkify-issues": "sindresorhus/linkify-issues#domify",
```
Comment (sindresorhus) [inline package.json]: This is out now.

**Comment by sindresorhus** [comment]
Can you fix the merge conflict?

**Comment by sindresorhus** [comment]
The URLs are not linkified here: https://github.com/sindresorhus/caprine/pull/220/files and the issue reference in the title is not linkified here: https://github.com/sindresorhus/refined-github/pull/299

**Pull request merged**